product_docs/docs/migration_toolkit/55/installing/macos.mdx

```diff
@@ -17,9 +17,11 @@ EDB provides a graphical interactive installer for MacOS. You can access it two
 
 You must have a Java JVM (version 1.8.0 or later) in place before Stack Builder can perform a Migration Toolkit installation. [Adoptium](https://adoptium.net) offers prebuilt OpenJDK binaries from fully open sources.
 
-You must also set the `JAVA_HOME` environment variable based on the location where you installed Java. For MacOS version 10.15 and later, you can use the following command to set the variable:
+You must also set the `JAVA_HOME` environment variable. Use the following command:
 
-`echo export "JAVA_HOME=\$(/usr/libexec/java_home)" >> ~/.zshrc`
+```shell
+JAVA_HOME=$(/usr/libexec/java_home)
```

Comment:
Should be

export JAVA_HOME=$(/usr/libexec/java_home)

(Otherwise it just forgets the env var immediatly)